tests/system/dub.json

```diff
@@ -5,7 +5,7 @@
     "dflags": [ "-i" ],
     "libs-posix": [ "sodium", "sqlite3" ],
     "libs-windows": [ "libsodium", "libsqlite3" ],
-    "versions": 
+    "versions":
```

Comment:
It looks like the space should be there.

Reply:
Nope, trailing whitespaces are bad (and I don't know why we have one here)